### PR TITLE
Issue 393 footer enhancement

### LINKED ISSUE #393: Enhancement: Update app footer
### Describe the Current Behavior/Feature:
Much of the app footer is currently a placeholder. For example, the "Proposal" button leads nowhere, the social media links are not properly connected, there's no tagline or logo, etc.

---
### Proposed Behavior/Feature:
Fill out all the missing information. Make sure it properly links to all relevant sites. Update the layout as well, if that is needed.

---
### Rationale:
Apps shouldn't have broken links or incomplete information. The user should be able to properly access everything that's presented to them.

---
### Proposed Implementation (if applicable):
Discuss with leadership to ensure what information should be included. Especially since it will link to terms and conditions as well as the privacy policy. Also, confirm the layout with the UX team.

---
### Additional context:
Add any other context or screenshots about the enhancement request here.

### PR DESCRIPTION
**Pull request by Raspber**
---
name: Pull Request
about: Create a pull request to merge your code into the PASS codebase
title: 'Enhance: app footer'
labels: 'enhancement, footer'
assignees: 'Raspber'

---

Enhance App Footer

---
## This PR:
**1.**    Updated "proposal" button to a call to action in the RenderCallToActionSection.jsx
**2.**    Updated Twitter icon to the newest X icon. 
**3.**    Removed unnecessary items within the RenderCompanyInfoSection.jsx, such as the codepdx logo with the tagline, and the built by. 

---
### Components

- Footer.jsx
- RenderCallToActionSection.jsx
- RenderCompanyInfoSection.jsx

### Tests
No test files are affected

### Other Files
No other files are affected

---
## Screenshots (if applicable):
No update
footer-no-update

Updated
footer-updated

---
## Additional Context (optional):
Will be creating a new issue to create new pages/routes for the terms and conditions link. 

---
## Future Steps/PRs Needed to Finish This Work (optional):

---

### CODE REVIEW

File: src/components/Footer/RenderCompanyInfoSection.jsx [hunk 1]
```diff
@@ -5,9 +5,20 @@ import Box from '@mui/material/Box';
...
 import Stack from '@mui/material/Stack';
 import Typography from '@mui/material/Typography';
-import TwitterIcon from '@mui/icons-material/Twitter';
+
 import FacebookIcon from '@mui/icons-material/Facebook';
 import InstagramIcon from '@mui/icons-material/Instagram';
+import SvgIcon from '@mui/material/SvgIcon';
+
+// new Twitter Icon
+const TwitterIcon = (props) => (
```
Comment (leekahung) [inline src/components/Footer/RenderCompanyInfoSection.jsx]: If this is simply a new custom component for the new Twitter Icon, I don't think we need to add a prop to this.

File: src/components/Footer/RenderCompanyInfoSection.jsx [hunk 2]
```diff
@@ -5,9 +5,20 @@ import Box from '@mui/material/Box';
...
+  <SvgIcon {...props} viewBox="0 0 512 512" style={{ width: '1.25em', height: '1.25em' }}>
```
Comment (leekahung) [inline src/components/Footer/RenderCompanyInfoSection.jsx]: With regards to the attributes, think we could use `sx` instead of `style` for MUI.

**Review by leekahung — CHANGES_REQUESTED** [review]
See comment

**Comment by Raspber** [comment]
> See comment

Will get this sorted 👍🏽

**Review by leekahung — APPROVED** [review]
Think this is good to go for merger. Given that it's fairly small, I think we could get this merged into Master as well along with PR #397 in the version update.

File: src/components/Footer/RenderCompanyInfoSection.jsx [hunk 3]
```diff
@@ -37,34 +48,19 @@ const socialLinks = [
...
-  >
+  <Stack width={isReallySmallScreen ? 1 : 1 / 5} justifyContent="space-between" alignItems="center">
     <Box>
-      <Typography color="tertiary.main">PASS LOGO</Typography>
-      <Typography color="#fff">tagline</Typography>
-    </Box>
-    <Box>
-      <Typography color="tertiary.main">Follow Us</Typography>
-      <Stack direction="row" spacing={1}>
+      <Typography color="tertiary.main" variant="h5" mb={1}>
```
Comment (xscottxbrownx) [inline src/components/Footer/RenderCompanyInfoSection.jsx:59]: Now the icons aren't as responsive and have issues at certain screen sizes.

Screenshot 2023-09-11 at 9 11 00 AM

File: src/components/Footer/RenderCallToActionSection.jsx [hunk 4]
```diff
@@ -25,7 +25,15 @@ const RenderCallToActionSection = ({ isReallySmallScreen }) => (
       If your organization is interested in partnering with PASS and would like to discuss further,
       contact us below.
     </Typography>
-    <Button variant="contained" color="secondary" sx={{ my: '1rem', width: 1 / 2 }}>
+    <Button
+      variant="contained"
+      aria-label="proposal-request"
+      color="secondary"
+      href="mailto:hugh@codeforpdx.org"
+      target="_blank"
```
Comment (xscottxbrownx) [inline src/components/Footer/RenderCallToActionSection.jsx:33]: not sure if I'm wrong from a security standpoint, but you don't need line 33 or 34 with a `mailto`, it opens in the email client not browser.

**Review by xscottxbrownx — CHANGES_REQUESTED** [review]
Overall good changes.
---
1) There's an issue with the icon spacing if you resize browser.
2) I would say go ahead and change the other footer file as well to make the copyright `CODE PDX` (dropping the "for" as we are rebranding, and all caps.)

**Comment by Raspber** [comment]
> Overall good changes.
> ---
> 1) There's an issue with the icon spacing if you resize browser.
> 2) I would say go ahead and change the other footer file as well to make the copyright `CODE PDX` (dropping the "for" as we are rebranding, and all caps.)

Will get this sorted out today.

**Comment by leekahung** [comment]
Hey @Raspber. What is the progress of this? I see that Scott pointed out some thing that needs to be addressed.

**Comment by Raspber** [comment]
> Hey @Raspber. What is the progress of this? I see that Scott pointed out some thing that needs to be addressed.

Working on this as well

**Comment by andycwilliams** [comment]
Since we're transitioning the organization name from Code for PDX to CODE PDX, we'll eventually need to update this again with the new branding. However, for this PR I think it's fine as is since we're not entirely sure when that whole process will be complete.

**Comment by Raspber** [comment]
ah sorry, i reguested review but i have yet to make the correct changes. please ignore.

**Review by xscottxbrownx — APPROVED** [review]

**Comment by ogorman89** [comment]
Nice 👍 

Like @xscottxbrownx said, the social icons get pretty crowded right before the breakpoint for mobile (just over ~600px viewport). I recommend we merge as is. 

There will be more work on the footer with the forthcoming updates from the UI team.

**Review by ogorman89 — APPROVED** [review]
Per conversation at the work session. This looks good to me to merge as is.

**Pull request merged**